.ci/checker-framework.groovy

```diff
@@ -435,17 +444,68 @@ class CheckerFrameworkError implements Comparable<CheckerFrameworkError> {
             return i
         }
 
-        i = getMessage() <=> other.getMessage()
-        if (i != 0) {
-            return i
+        if (this.isUnstable() || other.isUnstable()) {
+            final String messageWithoutLineNumber = getMessage().replaceAll('\\d+', '')
+            final String thatMessageWithoutLineNumber = other.getMessage().replaceAll('\\d+', '')
```

Comment:
Will numbers that were an issue always be 3 digits? Maybe consider changing the regexp to just match 3 digit numbers.
Also why not match the `#` too?

Reply:
> Will numbers that were an issue always be 3 digits?

No, they are not, see the updated issue description, we have entries like `temp-var-20910`

>  Also why not match the # too?

Not possible as we have entries like `temp-var-20910`, also if we could do it, there is no point in doing it, would increase the complexity of regex unnecessarily.